vancouver-drsm.csl

```diff
@@ -0,0 +1,466 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="fr-FR">
+  <info>
+    <title>Vancouver DRSM (French)</title>
```

Comment:
So we try to avoid acronyms at almost all cost in titles, and, if an organization/journal only has one type of citation format, we try to avoid mentioning the type (Vancouver here) as well.

Would `<title>Directions Régionales du Service Médical (French)</title>` be a good title? (is DRSM the name of your company?)

Once we decide on a proper title for the style, we'd want the file name, style ID, and "self" link to match as well (e.g. to "directions-regionales-du-service-medical.csl" and "http://www.zotero.org/styles/directions-regionales-du-service-medical")